docs/index.rst

```diff
@@ -92,22 +92,12 @@ We've divided up the documentation into four sections with different purposes:
 
             To the API reference
 
-.. toctree::
-    :hidden:
-    :caption: Tutorials
-
-    All Tutorials <tutorials/index>
-    tutorials/intro
-    tutorials/getting_started
-    Calibrations <tutorials/calibrations>
-    Data Processor <tutorials/data_processor>
-    Curve Analysis <tutorials/curve_analysis>
-    Visualization <tutorials/visualization>
-    Custom Experiments <tutorials/custom_experiment>
-
 .. toctree::
     :hidden:
```

Comment:
Should this be hidden? Another option is to set `:maxdepth: 1` or 2.

Reply:
I think it's supposed to be hidden or the entire ToC gets rendered after the page contents:
![image](https://github.com/Qiskit-Extensions/qiskit-experiments/assets/3870315/42a2235a-c35b-4a2d-9241-7106e405131b)

This toctree is only used to register those links for the sidebar.